services/web/client/source/class/qxapp/data/Permissions.js

```diff
@@ -97,7 +97,8 @@ qx.Class.define("qxapp.data.Permissions", {
     getLowerRoles(role) {
```

Comment:
Shouldn't you just normalize the role names when loaded the first time instead of upper/lowering it at every call?

Reply:
why is the case of the roles even a problem? I would make sure it is consistent ... do not allow for wrong case